nss-day-cohort-44/rare-rest-api-malicous-vortices
# when retrieving a specific post, posts now have a list of key/value p…

…airs for reactions that include the reaction name as the key and total number of that reaction received as the value.

# ISSUE TICKET 
## Description


## Type of change
# Testing Instructions
`git fetch origin `
`git checkout ` 

Use a retrieve method on a specific post (http://localhost:8000/posts/1) and it should return a property called reaction_numbers that has key values of reaction : number of that reaction.

User Instructions: 

# Checklist:
